doc/sdk-cli-reference.md

```diff
@@ -178,25 +178,27 @@ Parent command for all OLM Catalog related commands.
 
 ### gen-csv
 
-Writes a Cluster Service Version (CSV) manifest and concatenated CRD files to `deploy/olm-catalog`.
+Writes a Cluster Service Version (CSV) manifest and optionally CRD files to `deploy/olm-catalog/{operator-name}/{csv-version}`.
 
 #### Flags
 
-* `--csv-version` (required) Semantic version of the CSV manifest.
-* `--csv-config` Path to CSV config file. Defaults to deploy/olm-catalog/csv-config.yaml.
+* `--csv-version` string - (required) Semantic version of the CSV manifest.
+* `--from-version` string - Semantic version of CSV manifest to use as a base for a new version.
+* `--csv-config` string - Path to CSV config file. Defaults to deploy/olm-catalog/csv-config.yaml.
+* `--update-crds` Update CRD manifests in deploy/{operator-name}/{csv-version} using the latest CRD manifests.
+* `--crds-dir` string - Dir containing CRD manifests. Only valid with `--update-crds`. Defaults to deploy/crds.
```

Comment:
As part of the csv config file you can specify the CRD/CR paths to use for csv generation. That defaults to `deploy/crd/...` when there's no config.

What happens if you have a different(non-default) CRD dir path in the csv config file, and you only pass `--update-crds`:
```console
$ cat deploy/olm-catalog/csv-config.yaml
crd-cr-paths: custom/path/to/crds/

$ operator-sdk olm-catalog gen-csv --csv-version 0.1.0 --csv-config=deploy/olm-catalog/csv-config.yaml --update-crds
```

Since `--crds-dir` defaults to `deploy/crds` will this update the CRD manifests in `deploy/{operator-name}/{csv-version}` from `deploy/crds`, but use `custom/path/to/crds/` for generating the CSV itself(specifically the [ownedCRDs](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/design/building-your-csv.md#owned-crds) `spec. customresourcedefinitions.owned`).

If so, I'm thinking we should probably respect the csv-config file and use the CRDs specified there.
Although it's a bit tricky allowing the crd-dir to be specified by both the config and `--crds-dir` file.

Reply:
I'll remove `--crds-dir` in favor of deferring to `csv-config.yaml`.